pkg/apis/crd/v1beta1/types.go

```diff
@@ -36,6 +36,7 @@ type AntreaAgentInfo struct {
 	LocalPodNum                 int32                       `json:"localPodNum,omitempty"`                 // The number of Pods which the agent is in charge of
 	AgentConditions             []AgentCondition            `json:"agentConditions,omitempty"`             // Agent condition contains types like AgentHealthy
 	APIPort                     int                         `json:"apiPort,omitempty"`                     // The port of antrea agent API Server
+	NodePortLocalRange          string                      `json:"nodePortLocalRange,omitempty"`          // The port range used by NodePortLocal
```

Comment:
It should be called `NodePortLocalPortRange`

Reply:
Good catch, I think it’s a minor change but reasonable. Do you have any comments about this? @edwardbadboy @tnqn

Reply:
I agree with Jianjun